immuneML/reports/encoding_reports/FeatureDistributionReport.py

```diff
@@ -0,0 +1,173 @@
+import warnings
+from pathlib import Path
+
+import plotly.express as px
+
+from immuneML.analysis.data_manipulation.DataReshaper import DataReshaper
+from immuneML.data_model.dataset.RepertoireDataset import RepertoireDataset
+from immuneML.reports.ReportOutput import ReportOutput
+from immuneML.reports.ReportResult import ReportResult
+from immuneML.reports.encoding_reports.EncodingReport import EncodingReport
+from immuneML.util.PathBuilder import PathBuilder
+
+
+
+class FeatureDistributionReport(EncodingReport):
+    """
+    Plots a boxplot for each feature of the encoded dataset in one of the two modes:
+    in the 'normal' mode there are normal boxplots corresponding to each column of the 
```

Comment:
since this report and FeatureValueBarplot are a bit similar, it would be nice if they can refer to each other in the docs. (saying: if you want to do xyz instead, see this other plot). Referencing is written like this: :ref:`FeatureValueBarplot`
When you do this, could you also update the reference in the docs of FeatureValueBarplot? it now references to FeatureValueDistplot, just replace that with the name for this report